azure-pipelines-master.yml

```diff
@@ -32,10 +32,28 @@ steps:
     arguments: '.\node_modules\gulp\bin\gulp.js master:build'
     workingFolder: tools/Deployment
 
+- task: EsrpCodeSigning@1
+  displayName: Sign executables and dlls
+  inputs:
+    ConnectedServiceName: 'CodeSigning-APEX'
+    FolderPath: '$(Build.SourcesDirectory)/target/Release'
+    signType: 'batchSigning'
+    batchSignPolicyFile: '$(Build.SourcesDirectory)/codesign_policy.json'
+    SessionTimeout: '60'
+    MaxConcurrency: '50'
+    MaxRetryAttempts: '5'
+
+- task: CmdLine@1
+  displayName: 'gulp pack'
+  inputs:
+    filename: node
+    arguments: '.\node_modules\gulp\bin\gulp.js master:pack'
+    workingFolder: tools/Deployment
+
 - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
```

Comment:
Is this task same with `EsrpCodeSigning@1`? Can we update to use the same one?

Reply:
Let me test with EsrpCodeSigning@1 and get back.